ModuleNotFoundError: No module named 'multiprocess'

### Issue Type

Build/Installation

### Installation Source

Source

### Secretflow Version

secrecflow 1.10.0b1

### Operating System and Distribution

linux ubuntu

### Python Version

3.10.13

### Bazel Version

_No response_

### GCC/Compiler Version

_No response_

### Issue Description

```shell
After installing secretflow using the commands:

conda create -n sf python=3.10
conda activate sf
pip install -U secretflow

When try to do tests using:
pytest --env sim -n auto -v --capture=no tests/xxx.py

Error is shown as below:

ImportError while loading conftest '/MYDIR/secretflow/tests/conftest.py'.
tests/conftest.py:30: in <module>
    import multiprocess
E   ModuleNotFoundError: No module named 'multiprocess'

However, multiprocess is installed (0.70.17), and 
'import multiprocess'
works well in python.
```


### Reproduction Code

```shell
See Description.
```


### Additional Context or Logs

_No response_

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -7,7 +7,7 @@ Please check [INSTALLATION.md](./docs/getting_started/installation.md) to build 
 
 Before start, please install required package for testing.
 ```sh
-pip install -r dev-requirements.txt
+pip install -r requirements.txt
 ```
 
 There are two types in secretflow/tests.

--- a/secretflow/data/horizontal/dataframe.py
+++ b/secretflow/data/horizontal/dataframe.py
@@ -54,7 +54,7 @@ class HDataFrame(DataFrameBase):
         >>> alice = PYU('alice')
         >>> bob = PYU('bob')
         >>> h_df = read_csv({alice: 'alice.csv', bob: 'bob.csv'},
-                            aggregator=PlainAggregagor(alice),
+                            aggregator=PlainAggregator(alice),
                             comparator=PlainComparator(alice))
         >>> h_df.columns
         Index(['sepal_length', 'sepal_width', 'petal_length', 'petal_width', 'class'], dtype='object')

